tests/unit/test_run_ml_pipeline_management_command.py

```diff
@@ -1,12 +1,14 @@
-# Tests for dummy ML pipeline
 from io import StringIO
+from unittest.mock import patch
 
 import pytest
 from django.core.management import call_command
 
 from consultation_analyser.consultations.models import Consultation
+from consultation_analyser.pipeline.backends.dummy_llm_backend import DummyLLMBackend
+from consultation_analyser.pipeline.backends.sagemaker_llm_backend import SagemakerLLMBackend
 
-
+@patch("consultation_analyser.pipeline.backends.sagemaker_llm_backend.SagemakerLLMBackend", DummyLLMBackend)
```

Comment:
This is quite spooky — maybe better to set the ENV var so USE_SAGEMAKER_LLM is false? as in https://github.com/i-dot-ai/consultation-analyser/blob/main/tests/commands/test_dummy_data.py#L34

Reply:
Actually have removed the `USE_SAGEMAKER_LLM` env variable and just using `LLM_BACKEND` - I think that's simpler overall.

And then that line just gets removed from this particular test.